### ISSUE TITLE
Youtube video position is not restored on AA

### ISSUE THREAD

**Issue by lprot** [AndreyPavlenko/Fermata#311]
If you have both, youtube and TV enabled and youtube is configured to autostart in fermata auto settings, when you turn off ignition and then turn on, when AA connection is restored, fermata auto plays last played TV link instead of restoring and playung the last youtube video.

**Comment by lprot** [comment]
Tested on 1.9.5. If any beta testing or any logging is needed, just let me know.

**Comment by AndreyPavlenko** [comment]
Resuming youtube playback is not currently implemented.

**Comment by lprot** [comment]
Can it be at least fixed that TV does not play automatically if youtube is selected as autoload in settings?

**Comment by AndreyPavlenko** [comment]
Yes, it could be fixed, but there is a quick fix - disable auto play in the AA settings.

**Comment by lprot** [comment]
I'm using AA wireless and sometimes listen Spotify and most of the time use Fermata Auto with Youtube. That is cool that AA autoplays the last use source. So disabling autoplay is kind of workaround that requires additional manual interraction.
It would be very cool if Fermata would restore not exact position but at least last played YT video. And if that is a long story to implement, then just not autoplaying TV instead of YT would already be very nice.
Anyway thanks for the great player!
